### DISCUSSION TITLE
Issue contracts

### DISCUSSION THREAD

**Discussion by Kingmidas74** [nclient/NClient#454] [Ideas]
I think we should modify issue templates for features. Maybe some strict mode like 

- Problem
- Description
- Example
- Info link

will be better than simple sentences.

**Comment by Kingmidas74** [comment]
@smolchanovsky

**Comment by smolchanovsky** [comment]
Templates for issues already exist in the repository. They are available for selection when creating issue

**Reply by Kingmidas74** [reply]
I mean **strict** rules. Many of our issues doesn't fit to template.
